src/amo/components/AddonDetail.js

```diff
@@ -3,7 +3,7 @@ import React, { PropTypes } from 'react';
 
 import AddonMeta from 'amo/components/AddonMeta';
 import InstallButton from 'core/components/InstallButton';
-import LikeButton from 'amo/components/LikeButton';
+/*import LikeButton from 'amo/components/LikeButton';*/
```

Comment:
> Is there a LikeButton component (and tests) we could remove as well? I don't know if it's worth removing or not, but if we aren't planning to use it for awhile and there isn't much code it's probably best.

@tofumatt I'd originally suggested leaving the component for v2 here https://github.com/mozilla/addons-frontend/issues/1295#issuecomment-256430071 but since that's going to break coverage we may as well just delete it all and get it from history if we need anything since it's just a stub anyway.